src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/Application/Queries/SearchApprenticeships/SearchApprenticeshipsQueryHandler.cs

```diff
@@ -62,6 +63,14 @@ public async Task<SearchApprenticeshipsResult> Handle(SearchApprenticeshipsQuery
             var categories = routes.Routes.Where(route => request.SelectedRouteIds != null && request.SelectedRouteIds.Contains(route.Id.ToString()))
                 .Select(route => route.Name).ToList();
 
+            var totalWageTypeVacanciesCount = new GetApprenticeshipCountResponse { TotalVacancies = 0 };
```

Comment:
Could this run in parallel with the api call to get the vacancies?

Reply:
Because this call is required when the sort type is by SalaryAsc or SalaryDesc, so it would be good to keep it conditional